src/three-components/ARRenderer.js

```diff
@@ -272,7 +269,12 @@ height: 100%;`);
       const hitMatrix = matrix4.fromArray(hit.hitMatrix);
 
       this.dolly.position.setFromMatrixPosition(hitMatrix);
-      this.dolly.rotation.set(0, -presentedScene.pivot.rotation.y, 0);
+
+      // Orient the dolly/model to face the camera
+      const angle = Math.atan2(
+          this.camera.matrix.elements[12] - this.dolly.position.x,
+          this.camera.matrix.elements[14] - this.dolly.position.z);
+      this.dolly.rotation.y = angle - presentedScene.pivot.rotation.y;
```

Comment:
Feels like this should be easy with something like:

```js
let target = camera.position;
this.dolly.lookAt( position.x, 0, position.z );
```

Reply:
@mrdoob that does not prevent the dolly rotating on its Y

Reply:
Oh, sorry...

```js
this.dolly.lookAt( camera.position.x, this.dolly.position.y, camera.position.z );
```

Reply:
```
this.dolly.lookAt(camera.position.x, this.dolly.position.y, camera.position.z);
this.dolly.quaternion.multiply(q.setFromAxisAngle(Y_AXIS, -presented.pivot.rotation.y));
```

For a 180 degree turn on the Y axis, `lookAt` produces X and Z with `-PI`, and the quaternion multiplication nudges the Y value. Is this preferable? Is this quaternion math correct? Tests would have to be updated for the quaternion forms as well.

Reply:
Would this work?

```js
this.dolly.lookAt(camera.position.x, this.dolly.position.y, camera.position.z);
this.dolly.rotateY(Math.PI);
```

Reply:
It'd end up with the same visual transformation yeah. Instead of a euler `(0, PI+offset, 0)` rotation, it'll be `(-PI, offset, -PI)`

Reply:
I was just trying to suggest a more readable code.

Reply:
@mrdoob I'm mostly just unsure about testing and being able to represent multiple visually similar rotations e.g. `-PI, 0, -PI` being the same as `0, PI, 0`, but I'll take a look

Reply:
Eh, no different than checking just the rotation anyway, and this isn't called terribly often, I'll make the change, thanks! @mrdoob

Reply:
This makes the test a bit confusing (at least to my understanding of rotations), but added a comment explaining it